invenio_records_lom/resources/serializers/schemas.py

```diff
@@ -342,11 +348,12 @@ def get_contributors(self, obj: LOMRecord):
         contributes = obj["metadata"].get("lifeCycle", {}).get("contribute", [])
         contributordicts = []
```

Comment:
minor: normally the type of the items is not placed in the name. contributors is enough.

Reply:
Noted.
OTOH, `contributors` and `contributes` are pretty similar variable names to have in the same scope...
I wished naming variables was less hard...

Reply:
naming variables is one of the hardest things for a developer ;)

if you think that there could be a confusion because of similar names then you should name it completely different. but i think that is not necessary in this context.

contributes and contributors is ok for me.

Reply:
I renamed it.